untitled1/src/main/com/aca/homework/week7/clickable/AbstractClickable.java

```diff
@@ -0,0 +1,13 @@
+package com.aca.homework.week7.clickable;
+
+public abstract class AbstractClickable implements Clickable {
+    private Action action;
+
+    public AbstractClickable(Action action) {
```

Comment:
why did  we separate AbstactClicable class, if we can do all this in Button and Image classes, what is the basic meaning of separating AbstactClickable class, the name Abstactclickable class  is "Abstract"clickable but it is not abstract, it is a bit confusing.